### PR TITLE
More text and typo fixes

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

File: aiidalab_qe/process.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class WorkChainSelector(ipw.HBox):
 
     def __init__(self, **kwargs):
         self.work_chains_prompt = ipw.HTML(
-            "<b>Select computed workflow or start a new:</b>&nbsp;"
+            "<b>Select computed workflow or start a new one:</b>&nbsp;"
```
Comment (unkcpz) [inline aiidalab_qe/process.py:32]: I have a close look, and there is one in Nicola's feedback, that overlaps with other text. My fault for not checking it carefully.

File: aiidalab_qe/pseudos.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ class PseudoFamilySelector(ipw.VBox):
     )
     dft_functional_help = ipw.HTML(
         """<div style="line-height: 140%; padding-top: 10px; padding-bottom: 10px; opacity:0.5;">
-        The exchange-correlation energy is calculated functional. We currently provide support for two
+        The exchange-correlation energy is calculated using this functional. We currently provide support for two
```
Comment (unkcpz) [inline aiidalab_qe/pseudos.py:46]: My careless about this.

**Review by csadorf — APPROVED** [review]
LGTM!

File: aiidalab_qe/steps.py [hunk 3]
```diff
@@ -169,7 +169,7 @@ class SmearingSettings(ipw.VBox):
         """<p>
         The smearing type and width is set by the chosen <b>protocol</b>.
         Tick the box to override the default, not advised unless you've mastered <b>smearing effects</b> (click <a href="http://theossrv1.epfl.ch/Main/ElectronicTemperature"
-        target="_blank">HERE</a> for a discussion).
+        target="_blank">here</a> for a discussion).
```
Comment (unkcpz) [inline aiidalab_qe/steps.py:172]: It all capitalized on Nicola's writing, I guess it is not necessary to use 'HERE', lowercase should be fine.

**Review by unkcpz — COMMENTED** [review]
Mostly from my carelessness.

**Pull request merged**